src/docfx/build/xref/InternalXrefSpec.cs

```diff
@@ -29,19 +29,36 @@ public string GetXrefPropertyValue(string propertyName)
 
         public string GetName() => GetXrefPropertyValue("name");
 
-        public ExternalXrefSpec ToExternalXrefSpec(Context context, Document file)
+        public ExternalXrefSpec ToExternalXrefSpec(Context context, Document file, bool forOutput = true)
```

Comment:
should we add `branch=master` regardless? the only different is whether to output relative url or absolute url

Reply:
do you mean we should append `?branch=master` to internal UID resolved URL as well? For internal UID, the relative URL should exist with the current branch, no need to fall back to `master`, right?